### ISSUE TITLE
Список сообществ-участников и их профилей

### ISSUE THREAD

**Issue by Leningradez** [Piter-United/itgm#8]
Как Участник ITGM, хочу ознакомиться со списком сообществ-участников и описание каждого из сообществ, чтобы выбрать интересные мне и присоединиться к их онлайн площадкам.

Примечание:
- Список сообществ с возможностью поиска
- Профиль сообщества с описанием
- Ссылки на все онлайн площадки сообщества (внешние ссылки)

Доп. возможности:
- В профиле сообщества, показывать список участников, выбравших сообщество как "основное" на этапе регистрации
- В профиле сообщества, показывать список тем, поданных и размеченных этим сообществом

**Comment by alexsuslov** [comment]
откройте доступ к макету

**Comment by Leningradez** [comment]
@alexsuslov задачу на себя ассайнь - если взялся за нее, доступ выдали.

**Comment by alexsuslov** [comment]
проблема что нет основного лайаут
header не поддерживает светлую  тему
в футоре есть иконки  vk и in ytn в наборе ant
не знаю у кого спросить где взять данные для столбиков в футоре
![Снимок экрана 2020-01-24 в 14 12 32](https://user-images.githubusercontent.com/3445074/73065029-9c442080-3eb3-11ea-9e79-c6bbc93b1df3.png)

**Comment by Leningradez** [comment]
да, лайаут @mike1pol обещал сделать! Данные в футтере захардкодим думаю
